service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -138,4 +142,71 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
         );
     }
 
+    @Override
+    public Optional<ItemFileMetadata> findItemFileMetadataByStimulusKey(final String clientName, final long bankKey, final long stimulusKey) {
```

Comment:
I went back and forth on modifying the existing query, but I hesitate to do that until all of the scoring stuff has shaken out.